1.0/docs/devguide/tests.md

```diff
@@ -0,0 +1,367 @@
+---
+layout: default
+type: guide
+shortname: Docs
+title: Unit tests
+subtitle: Developer guide
+---
+
+<link rel="import" href="../../../0.5/components/google-youtube/google-youtube.html">
+
+{% include toc.html %}
+
+Use Web Component Tester (`wct`) to unit test your Polymer elements. 
+
+`wct` is an end-to-end testing environment built by the Polymer core team. It
+enables you to test your elements locally, against all of your installed
+browsers, or remotely, via Sauce Labs. `wct` is composed of the following
+tools:
+
+*   [Mocha](https://mochajs.org/) for test framework, complete with support 
+    for BDD and TDD.
+*   [Chai](http://chaijs.com/) for assertions. 
```

Comment:
Is it worth adding for Chai and Sinon that they're adding optional extras. That is, something like:

Chai: adds more assertion types that can be used with Mocha tests.
Sinon: adds test spies, stubs and mocks.

The Sinon one feels extra awkward because I don't expect everyone to know what test spies, stubs and mocks are in the first place. Open to suggestions here.